packages/Webkul/Shop/src/Resources/lang/en/app.php

```diff
@@ -840,6 +840,16 @@
                 'sign-in'     => 'Sign in',
             ],
 
+            'admin' => [
+                'new_customer_registration_notification' => [
+                    'dear' => 'Dear Admin',
+                    'thanks' => 'Thank you.',
+                    'description' => 'A new customer has registered in the system.',
+                    'name' => 'Customer Name: :customerName',
+                    'subject' => 'New Customer Registration',
+                ],
+            ],
```

Comment:
Please add the proper indentation here. Like spacing. And arranging the keys based on alphabetical order, starting with the first letter.